integration_tests/views/async_view.py

```diff
@@ -0,0 +1,11 @@
+def AsyncView():
```

Comment:
Shouldn't this be `snake_case` as it is a function? Same for `SyncView`. Now that I think about it, defining views with a class could be cool too?

Reply:
> Shouldn't this be snake_case as it is a function? 

Actually, no. I wanted the views to be easily identifiable. Having class views is something up for the future, but implementing this as functional components was more difficult and interesting 🤣